test/io-tests.sh

```diff
@@ -43,7 +43,7 @@ pgrStart(){ postgrest $1 >/dev/null 2>/dev/null & pgrPID="$!"; }
 pgrStartRead(){ postgrest $1 <$2 >/dev/null & pgrPID="$!"; }
 pgrStartStdin(){ postgrest $1 >/dev/null <<< "$2" & pgrPID="$!"; }
 pgrStarted(){ kill -0 "$pgrPID" 2>/dev/null; }
-pgrStop(){ kill "$pgrPID" 2>/dev/null; pgrPID=""; }
+pgrStop(){ kill "$pgrPID" 2>/dev/null; pgrPID=""; sleep 0.1; }
```

Comment:
This should fix the "segmentation fault" problem we had with stack-test as well. Not relevant anymore, because we removed the io-tests there, just noting.

Had a similar problem. Once coverage is enabled, the coverage report file from process A is not closed yet when process B already tries to write to it. This leads to the bash shell running the io-tests file erroring out with segmentation fault. In the case of stack-test I mostly observed it when the port process B tried to liste on was not closed yet from process A.